lib/Remote/RemoteTargetLocator.php

```diff
@@ -68,10 +68,22 @@ public function defaultContent()
      */
```

Comment:
Null should be mentioned in `@param` - and also with explanation what each value means (WebDriverElement/null/numeric id)